.azure-pipelines/templates/jobs/packaging-jobs.yml

```diff
@@ -63,22 +63,47 @@ jobs:
       matrix:
         amd64:
           ARCH: amd64
-        arm64:
-          ARCH: arm64
-        armhf:
-          ARCH: armhf
+        # Do not run the QEMU jobs for the full-test-suite pipeline
+        ${{ if ne(variables['System.DefinitionId'], 4 )}}:
```

Comment:
Like you flagged in your initial post, I think we probably should run this for `*.x` release branches. I think this is unlikely, but I think it's possible that there is a problem that only occurs when building the snap for another architecture and I'd hate for this to only be found after the point release rather than before.

This applies to us not building the other architectures below too.

What if we just skipped this on test branches with:
```
${{ if not(startsWith(variables['Build.SourceBranchName'], 'test-')) }}
```

Reply:
Indeed. I switched to this condition.